project_task_material/__manifest__.py

```diff
@@ -21,6 +22,6 @@
     ],
     "data": [
         "views/project_view.xml",
-        "security/ir.model.access.csv",
-    ],
+        "security/ir.model.access.csv"
```

Comment:
Why do you remove commas? Don't touch what is not needed. And your style is worse then the previous one.